audio/FAudio/Portfile

```diff
@@ -18,12 +18,23 @@ long_description        an XAudio reimplementation that focuses solely on develo
 
 depends_lib-append      port:libsdl2
 
-checksums               rmd160  c3a00ea95ac9f34e5011ba1aa1b4553f47b0526c \
-                        sha256  654500c4adac47fdd4885877c2496d8b4c313af7a6b59f7cfca76e03297d28f6 \
-                        size    908871
-
-# remove set deployment target
-patchfiles              patch-faudio-remove-deployment-target.diff
+checksums               rmd160  b22854b0b36c1041b68bf1edb0fdaea4b042bfa4 \
+                        sha256  92cafeaf7b61e6165f632c0d4564704b9cf326525639298eedc3ae2454a2bf3c \
+                        size    909352
+
+# Bump macOS minimum to 10.9, per cert requirements
+# https://github.com/FNA-XNA/FAudio/commit/30b692372a81189cf57f013e8252e53edd57dcf1
+# So fallback to 20.3 for systems below 10.9
+if {${os.major} < 13} {
+    github.setup            FNA-XNA FAudio 20.03
+    epoch                   1
```

Comment:
The `epoch` line is not necessary here because the current version is 20.03.

Reply:
Removed, I was sure I removed that before making that commit